AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/ChangeVideoPlayerActivity.java

```diff
@@ -37,6 +38,9 @@ public final static String getName() {
   final String PCODE  = "R2d3I6s06RyB712DN0_2GsQS-R-Y";
   final String DOMAIN = "http://ooyala.com";
 
+  // Write the sdk events text along with events count to log file in sdcard if the log file already exists
+  SDCardLogcatOoyalaEventsLogger Playbacklog= new SDCardLogcatOoyalaEventsLogger();
```

Comment:
sure will do that

On Tue, Mar 8, 2016 at 1:58 PM, lental <notifications@github.com> wrote:

> In
> AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/ChangeVideoPlayerActivity.java
> <https://github.com/ooyala/android-sample-apps/pull/71#discussion_r55435349>
> :
>
> > @@ -37,6 +38,9 @@ public final static String getName() {
> >    final String PCODE  = "R2d3I6s06RyB712DN0_2GsQS-R-Y";
> >    final String DOMAIN = "http://ooyala.com";
> >
> > +  // Write the sdk events text along with events count to log file in sdcard if the log file already exists
> > +  SDCardLogcatOoyalaEventsLogger Playbacklog= new SDCardLogcatOoyalaEventsLogger();
>
> can you add a space before the equals, and lowercase the "P" in
> playbackLog?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/android-sample-apps/pull/71/files#r55435349>.
>



-- 
Regards,

BinduMadhav Sondur
310, Crescent Village Circle , #2110 ,
San Jose , CA - 95134

PH: 315-247-5338